setup.py

```diff
@@ -42,10 +42,10 @@ def get_version(version_file):
 
 
 setup(
-    name="example",
+    name="pe_reports",
     # Versions should comply with PEP440
-    version=get_version("src/example/_version.py"),
-    description="Example Python library",
+    version=get_version("src/pe_reports/_version.py"),
+    description="Posture and Exposure Report's library",
```

Comment:
If the apostrophe is intended to denote plurality instead of ownership then it should be deleted:
```suggestion
    description="Posture and Exposure Reports library",
```

If it is meant to denote ownership then ignore this comment.

Reply:
Resolved in 4002c20

Reply:
Thanks @schmelz-ctr for committing this suggestion.  In the future you might try making the commit message a little more descriptive and specific.  Even the bland and generic "Fix typo" is better than "Update setup.py".